qa/tasks/cephfs/fuse_mount.py

```diff
@@ -424,20 +425,27 @@ def find_socket(client_name):
         for f in files:
                 pid = re.match(".*\.(\d+)\.asok$", f).group(1)
                 if os.path.exists("/proc/{{0}}".format(pid)):
-                        return f
+                    with open("/proc/{{0}}/cmdline".format(pid), 'r') as proc_f:
+                        if mountpoint in proc_f.read():
+                            return f
         raise RuntimeError("Client socket {{0}} not found".format(client_name))
 
-print(find_socket("{client_name}"))
+print(_find_socket("{client_name}"))
 """.format(
             asok_path=self._asok_path(),
-            client_name="client.{0}".format(self.client_id))
+            client_name="client.{0}".format(self.client_id),
+            mountpoint=self.mountpoint)
 
         # Find the admin socket
         asok_path = self.client_remote.sh(
             ['sudo', 'python3', '-c', pyscript],
             stdout=StringIO(),
             timeout=(15*60)).strip()
```

Comment:
Here let's `s/self.client_remote.sh()/self.run_python()/`, and the `run_python` could work well for me locally.

Reply:
Done.